Engine/csound_standard_types.c

```diff
@@ -385,12 +403,16 @@ const CS_TYPE CS_VAR_TYPE_ARRAY = {
   array_free_var_mem, NULL, 0
 };
 
-
 const CS_TYPE CS_VAR_TYPE_INSTR = {
   "InstrDef", "instrument definition reference", CS_ARG_TYPE_BOTH,
   createInstrRef, instrRef_copy_value, NULL, NULL, 0
 };
 
+const CS_TYPE CS_VAR_TYPE_INSTANCE = {
```

Comment:
The name CS_VAR_TYPE_INSTANCE seems very generic, suggest this changes to CS_VAR_TYPE_INSTR_INSTANCE.

Reply:
done